### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by ghost**
Set up the db_build.js, db_build.sql and db_connect.js files.
Set up the schemas according to #4  
added test user to user database
Added a db to our heroku and linked it in config.env
added config.env to .gitignore
relates #4 #34

### CODE REVIEW

**Review by joeylouise — APPROVED** [review]

File: src/database/db_connect.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { Pool } = require('pg');
+const url = require('url');
+
+const environment = require('env2');
+
+environment('./config.env');
+
+if (!process.env.DATABASE_URL) {
+  throw new Error('DATABASE_URL missing from .env.');
```
Comment (des-des) [inline src/database/db_connect.js:9]: :tada:

**Review by des-des — APPROVED** [review]
wikid

**Comment by des-des** [comment]
@ConchobarMacNessa resolve conflicts then re-assign!

**Comment by des-des** [comment]
@ConchobarMacNessa also if you want to fix spacing in sql file that would be cool but not a blocker

**Pull request merged**